File: LatB/Php/Pc.php
<?php
class Pc{
    #attribut dari class yang telah dibuat dan total harga (dimana total harga didapat dari penjumlahan harga dari semua class)
    private $CPU;
    private $RAM;
    private $DISK;
    private $total_price;
    #constructor dimana class composition akan dipanggil 
    function __construct(Ram $RAM, Cpu $CPU, Disk $DISK, $total_price)
    {
        $this->total_price = $total_price;
        $this->RAM = $RAM;
        $this->CPU = $CPU;
        $this->DISK = $DISK;
    }
    #Get dan Set untuk setiap attribute private
    function setCpu(Cpu $CPU){
        $this->CPU = $CPU;
    }
    function getCpu(){
        return $this->CPU;
    }
    function setRam(Ram $RAM){
        $this->RAM = $RAM;
    }
    function getRam(){
        return $this->RAM;
    }
    function setDisk(Disk $DISK){
        $this->DISK = $DISK;
    }
    function getDisk(){
        return $this->DISK;
    }
    function setTotal_price($total_price){
        $this->total_price = $total_price;
    }
    function getTotal_price(){
        return $this->total_price;
    }
    #fungsi untuk menampilkan isi dari semua class/attribute private
    function printPc(){
        echo "Spesifikasi PC Mang Ujang" ."<br>";
        $this->CPU->printCpu();
        $this->RAM->printRam();
        $this->DISK->printDisk();
        echo "Total Price: " .$this->total_price."<br>";
    }
    function __destruct()
    {
        
    }
}
?>

File: LatB/Java/Pc.java
public class Pc{
    // Attribut untuk class Pc
    // Class-class yang telah dibuat dijadikan sebagai composition atau attribut class Pc
    private Ram RAM;
    private Cpu CPu;
    private Disk DISK;
    private int total_price;
    // Constructor
    Pc(){}
    //class dan attribute yang telah dibuat dijadikan sebagai parameter
    public Pc(Ram rAM, Cpu cPu, Disk dISK, int total_price) {
        RAM = rAM;
        CPu = cPu;
        DISK = dISK;
        this.total_price = total_price;
    }
    // Get dan Set
    public Ram getRAM() {
        return RAM;
    }
    public void setRAM(Ram rAM) {
        RAM = rAM;
    }
    public Cpu getCPu() {
        return CPu;
    }
    public void setCPu(Cpu cPu) {
        CPu = cPu;
    }
    public Disk getDISK() {
        return DISK;
    }
    public void setDISK(Disk dISK) {
        DISK = dISK;
    }
    public int getTotal_price() {
        return total_price;
    }
    public void setTotal_price(int total_price) {
        this.total_price = total_price;
    }
    // Fungsi untuk menampilkan isi dari attribute dan composite
    public void printPc(){
        System.out.println("Spesifikasi Komputer Mang Ujang: ");
        RAM.printRam();
        CPu.printCpu();
        DISK.printDisk();
        System.out.println("Total Price: " + this.total_price);
    }
}